src/assets/scss/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$main_color: #000;

```

Comment:
Kindly manage the variables in a way that in future if we want to apply the dark/light modes in the app then we can easily add that functionality in the app by simply inversing the variable values.